Build/Remotion.Server.build

```diff
@@ -89,7 +89,7 @@
     <Error Text="The property 'BuildNumber' is not set." Condition="'$(BuildNumber)' == ''" />
 
     <PropertyGroup>
-      <Version>$(Version)-x.1.$(BuildNumber)</Version>
+      <Version>$(Version).$(BuildNumber)</Version> <!-- I had to change it since the old way threw an error -->
```

Comment:
The BuildScript has a check if the version is in a correct SemVer format as per http://semver.org/spec/v2.0.0.html, and adding the `-x.1.$(BuildNumber)` seems to break the pattern as "2.0.0-alpha.22-x.1.1337" is not correct SemVer.

Reply:
Yes, there's a way to deal with this. the version.props must not have "-alpha.20" in the entry.
https://github.com/re-motion/SharedSource-Build/blob/develop/Build/Customizations/Version.props#L12

This is also an issue when doing a no-prerelease patch of the version.props file. The current logic adds the prerlease info to the version number. => please revert.